spec/features/jobs/user_edits_a_job_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe "user edits an existing job" do
+  scenario "a user can edit a job" do
+    company = Company.create!(name: "StackOverflow")
+    job = company.jobs.create!(title: "Rails Developer",
+      description: "do rails things", level_of_interest: 3, city: "Denver")
```

Comment:
If you're going to break to a new line, I would likely break each element to a new line and indent so that they line up.